server/manager/src/main/java/org/apache/accumulo/manager/compaction/queue/CompactionJobQueues.java

```diff
@@ -91,6 +91,11 @@ public long getQueueCount() {
     return priorityQueues.mappingCount();
   }
 
+  public long getQueuedJobCount() {
+    return priorityQueues.values().stream().map(CompactionJobPriorityQueue::getQueuedJobs)
+        .reduce(Long::sum).orElseThrow();
```

Comment:
Instead of throwing an NoSuchElement exception, can we just return zero instead?
```suggestion
        .reduce(Long.valueOf(0), (Long::sum));
```